jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/profiles/ProfileCredentialProviderFactoryTest.kt

```diff
@@ -745,6 +745,32 @@ class ProfileCredentialProviderFactoryTest {
         }
     }
 
+    @Test
+    fun `sso-session profile creates a provider`() {
```

Comment:
needs tests for the case where the configuration is invalid/incomplete

Reply:
Added those tests